src/tracing/defaults.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Default tracing options
+ */
+export default {
+  enabled: false,
+  endpoint: 'api.rollbar.com/api/1/session/',
```

Comment:
Wonder if we should just define `/api/1/session/` and leave the host out. For example if we want to test this in our localhost? I assume the host is set in some other global config place?

Reply:
Let's simplify transport options in general in a follow up PR.